src/components/core/tests/PivotTable.spec.tsx

```diff
@@ -148,7 +148,7 @@ describe('PivotTable', () => {
                         uri: '/gdc/md/xms7ga4tf3g3nzucd8380o2bev8oeknp/obj/2011'
                     },
                     id: 'year',
-                    title: 'default (Date)'
+                    title: 'Quarter (Date)'
```

Comment:
New code is correct. Old one is wrong. Pivot Table did not worked with attributeHeaderItems same as original Table. Now it works same as original table - uses `attributeHeaderItems.formOf.name` as drilling title instead of `attributeHeaderItems.name`.